scripts/data_ingest/main.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import logging.config
+import os
+import sys
+from http import HTTPStatus
+from pathlib import Path
+from pprint import pprint
+
+import requests
+
+ADMIN_EMAIL_ENV = "SUPERUSER_EMAIL"
```

Comment:
Integrating these via an .env file?

Reply:
I'll update the README with execution instructions instead...